react-native/sample_integration/sample_integration/App.js

```diff
@@ -17,18 +17,14 @@ const App = () => {
   });
   const [checkedItems, checkedItemChange] = useState([]);
 
-  // EMBRACE HINT:
-  // Storing constants like moment or breadcrumb names make typos less likely.
-  const mountedMoment = 'App Mounted';
-
   useConstructor(() => {
     //This only happens ONCE and it happens BEFORE the initial render.
 
     // EMBRACE HINT:
-    // Moments are a great way to measure the performance and abandonment of workflows within your application.
-    // We recommend to wrap non user interruptable flows with moments. For example, here we are measuring how long
-    // it takes this component to render.
-    startMoment(mountedMoment);
+    // The Embrace SDK automatically records the special “startup” moment that’s used to track app launch performance.
+    // You should end the startup moment before the user has a chance to interact with the application.
+    // Add this method call to every location where the startup moment can end. You can call this method as many times as you like.
+    endAppStartup();
```

Comment:
We should call this in useEffect though like you had before, since that's when users will actually be able to see that the component loaded right?